docs/react/adapters/solid-query.md

```diff
@@ -161,7 +161,9 @@ const queryClient = new QueryClient()
 
 function Example() {
   const [enabled, setEnabled] = createSignal(false)
-  const query = createQuery(() => ['todos'], fetchTodos, {
+  const query = createQuery({ 
+    queryKey: () => ['todos'],
+    queryFn: fetchTodos,
     // ❌ passing a signal directly is not reactive
     // enabled: enabled(),
```

Comment:
I think this workaround is also unnecessary without the overloads. @ardeora can you have a look please